sites/main-site/src/content/blog/2024/tools-3_0_0.md

```diff
@@ -0,0 +1,252 @@
+---
+title: 'nf-core/tools - 3.0.0'
+subtitle: 'What flavour of the nf-core template do you prefer?'
+headerImage: 'https://images.unsplash.com/photo-1583258292688-d0213dc5a3a8'
+headerImageAlt: 'Yellow and red apples on black plastic crate'
+pubDate: 2024-10-03T12:00:00+01:00
+authors:
+  - 'mirpedrol'
+  - 'mashehu'
+label:
+  - 'tools'
+---
+
+# Highlights
+
+## ⛓️‍💥 Breaking changes
+
+- All pipeline commands now require the `pipelines` prefix.
+  For example, `nf-core lint{:bash}` is now `nf-core pipelines lint{:bash}`.
+  This change makes the commands more consistent with `nf-core modules{:bash}` and `nf-core subworkflows{:bash}` commands.
+- Some options have been changed for the `nf-core pipelines download{:bash}` command:
+  - The `-t` / `--tower` flag has been renamed to `-p` / `--platform`.
+  - We renamed the short flags for consistency, to always use the first letter of the second word in the long flag:
+    - The `-d` / `--download-configuration` flag has been renamed to `-c` / `--download-configuration`.
+    - The `-p` / `--parallel-downloads` flag has been renamed to `-d` / `--parallel-downloads`.
+
+## ✨ New features
```

Comment:
Maybe lead with this? Get people excited before slamming into the work we're asking them to do.